chart/values.yaml

```diff
@@ -64,6 +64,9 @@ ingress:
   # configurationSnippet: |
   #   more_set_input_headers "X-Forwarded-Host: {{ .Values.hostname }}";
 
+  # ingressClassName can be set to specify which IngressClass will be used by the created Ingress
+  ingressClassName: ""
```

Comment:
Change this to `className` so when it is used it will be `ingress.className` and won't "stutter"

Reply:
Sure thing!